test/models/location_test.rb

```diff
@@ -96,12 +96,13 @@ class LocationTest < ActiveSupport::TestCase
     assert mock.verify
   end
 
-  test "should find existing Location by API ID" do
+  test "should find existing Location by matching fields" do
+    location_info = { name: "Nevada, USA", geo_level: "state", country_name: "USA", state_name: "Nevada", subdivision_name: "", city_name: "" }
```

Comment:
Should we write a test for not yet normalized fields?

Reply:
Discussed offline: the existing fields would still need manual or migration correction unless we also wanted to add the normalization to `find_with_fields`